objects/ogamemanager/Step_0.gml

```diff
@@ -76,6 +76,42 @@ switch (global.gameMode)
 				}
 			}
 		}
+		
+		// When a player meets the win requirement for the match return players to the character selection screen
+		if (global.p1Rounds == 2 || global.p2Rounds == 2)

```

Comment:
This needs to be rewritten so the same condition isn't checked twice. You'll have to repeat 2 more lines, but it'll be more efficient with cpu resources.

```
if (global.p1Rounds == 2)
{
   global.gameHalt = true;
   gameHaltTimer++;
   var particle = instance_create_layer(global.camObj.x - 80, 0, "KO_Text", oParticles);
   //... etc
}
else if (global.p2Rounds == 2)
{
   global.gameHalt = true;
   gameHaltTimer++;
   var particle = instance_create_layer(global.camObj.x - 80, 0, "KO_Text", oParticles);
    // etc...
}
```

Alternatively, you could pull everything out into a function and just have the if call the function to reduce code repeat.

Something like this:

```
if (global.p1Rounds == 2)
{
    gameHaltTimer++;
    PlayerWins(1);
}
else if (global.p2Rounds == 2)
{
    gameHaltTimer++;
    PlayerWins(2);
}

// function
function PlayerWins(playerNumber)
{
    global.gameHalt = true;
    ...
    sprite_index = asset_get_index("sPlayer" + playerNumber + "Wins");
}
```